pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -48,8 +48,11 @@ const (
 	DeployerRoleName          = "system:deployer"
 	RouterRoleName            = "system:router"
 	RegistryRoleName          = "system:registry"
+	NodeRoleName              = "system:node"
+	SDNReaderRoleName         = "system:sdn-reader"
+	SDNManagerRoleName        = "system:sdn-manager"
```

Comment:
@rajatchopra does this split make sense (sdn-reader, sdn-manager)? Does the osdn plugin that runs in the node/master only do read operations?